docs/cudf/source/groupby.md

```diff
@@ -33,6 +33,23 @@ import cudf
 >>> gb3 = df.groupby(cudf.Series(['a', 'a', 'b', 'b', 'b']))  # grouping by an external column
 ```
 
+> :warning: **Important**: cuDF uses `sort=False` by default to achieve better performance
```

Comment:
Some how `:warning:` is not rendered on cudf page. I'm tending to try out the warning block.

Reply:
You can use the unicode character ⚠️instead. I think `:warning:` is a GitHub specific thing

Reply:
TBH, it looks so much better:
![image](https://user-images.githubusercontent.com/13521008/105553816-fb04cb80-5cba-11eb-85cf-df0666851ecf.png)

Reply:
Agree this looks much nicer. I think we should put the example also inside the warning box - is that possible?

Reply:
Yup!
![image](https://user-images.githubusercontent.com/13521008/105749962-fcbbd280-5ef8-11eb-8b93-668a0a78dd67.png)

Reply:
Awesome!